tests/test_integration.py

```diff
@@ -1,22 +1,27 @@
+from pathlib import Path
 import os.path as op
 import shutil
 
 import pytest
+import cython_setuptools
 
 
 this_dir = op.dirname(__file__)
+cython_setuptools_path = (Path(cython_setuptools.__file__).parent).parent
```

Comment:
Stay consistent, use existing `op` in this commit
`cython_setuptools_path = op.dirname(this_dir)` (no need to import `cython_setuptools`)
or
`cython_setuptools_path = op.abspath(op.join(cython_setuptools.__file__, "..", ".."))`

If you really want to move to `pathlib`, it should be a dedicated commit changing the whole module.

Reply:
use `op` 👍